src/containers/Storage/DiskStateProgressBar/DiskStateProgressBar.scss

```diff
@@ -6,6 +6,7 @@
     $inner-border-radius: $outer-border-radius - $border-width;
 
     position: relative;
+    z-index: 0;
```

Comment:
Set zero `z-index` to create stacking context, so progress bar text with high `z-index` won't overlay parent container context